### DISCUSSION TITLE
Can peacock export a animation of a .e file?

### DISCUSSION THREAD

**Discussion by jackeggemeyer** [idaholab/moose#26841] [Q&A Getting Started] [answered]
Can Peacock on Linux export/save an mp4/video file of my results(outputted as a .e file)? I am currently dual-booting and don't want to transfer my .e files onto Windows to load into Paraview. I am trying to avoid installing Paraview on Linux, as I have tried for a day and gave up.

**Comment by milljm** [comment, marked as answer]
I am sorry, but no... Peacock does not have this capability.

As for Paraview giving you grief on Linux, perhaps we can help with that? Honestly, I usually hear the opposite: Paraview works, but Peacock does not.

Paraview exists as a Conda package, if that helps? This is how I install it on my own machines (after installing the MOOSE Conda packages):
```bash
conda create -n paraview paraview
conda activate paraview
paraview
<eventually pops up on the screen>
```
If the above does not work, yet Peacock is "popping up" on the screen, I really can't fathom why Paraview would not.

**Reply by jackeggemeyer** [reply]
Thank you, didnt know conda had Paraview as a package
